src/main/java/com/commercetools/sync/categories/CategorySyncOptionsBuilder.java

```diff
@@ -46,8 +46,8 @@ public CategorySyncOptions build() {
     }
 
     /**
-     * Returns an instance of this class to be used in the super class generic methods. Please see the JavaDoc in the
-     * overridden method for further details.
+     * Returns an instance of this class to be used in the superclass's generic methods. Please see the JavaDoc in the
```

Comment:
```suggestion
     * Returns an instance of this class to be used in the superclass' generic methods. Please see the JavaDoc in the
```

Reply:
actually this was correct like that @lojzatran @ahmetoz https://www.grammarbook.com/punctuation/apostro.asp